client/mysqldump.c

```diff
@@ -784,7 +787,7 @@ static void write_header(FILE *sql_file, const char *db_name)
 
     if (!opt_logging)
       fprintf(sql_file,
-"\n/*M!100101 SET LOCAL SQL_LOG_OFF=0, LOCAL LOG_SLOW_QUERY=0 */;");
+"\n/*M!100101 SET LOCAL SQL_LOG_OFF=0, LOCAL SLOW_QUERY_LOG=0 */;");
```

Comment:
A comment on this change should be in your commit message.

Reply:
I was not supposed to change this line of code. I am going to check why it happened

Reply:
`LOG_SLOW_QUERY` does seem to the [unaliased one](https://mariadb.com/kb/en/server-system-variables/#slow_query_log)

Reply:
I really did not write this. The code from the current version 10.11 already brought SLOW_QUERY_LOG and I did not change that. I do not know why this is reporting that I changed from LOG_SLOW_QUERY, which was not even in the source code on which I was based.